java/serving-client/src/test/java/dev/feast/FeastClientTest.java

```diff
@@ -86,7 +87,7 @@ public void setup() throws Exception {
     ManagedChannel channel =
         this.grpcRule.register(
             InProcessChannelBuilder.forName(serverName).directExecutor().build());
-    this.client = new FeastClient(channel, Optional.empty());
+    this.client = new FeastClient(channel, Optional.empty(), Deadline.after(200, TimeUnit.MILLISECONDS));
```

Comment:
nit: can you make this a field defined at the top of the class?

Reply:
✅